src/server/game/Spells/SpellInfo.h

```diff
@@ -197,12 +197,13 @@ enum SpellCustomAttributes
     SPELL_ATTR0_CU_NO_POSITIVE_TAKEN_BONUS       = 0x00200000,
     SPELL_ATTR0_CU_SINGLE_AURA_STACK             = 0x00400000, // pussywizard
     SPELL_ATTR0_CU_SCHOOLMASK_NORMAL_WITH_MAGIC  = 0x00800000,
-    SPELL_ATTR0_CU_ENCOUNTER_REWARD              = 0x01000000, // pussywizard
+    SPELL_ATTR0_CU_AURA_CANNOT_BE_SAVED          = 0x01000000,
     SPELL_ATTR0_CU_POSITIVE_EFF0                 = 0x02000000,
     SPELL_ATTR0_CU_POSITIVE_EFF1                 = 0x04000000,
     SPELL_ATTR0_CU_POSITIVE_EFF2                 = 0x08000000,
-    SPELL_ATTR0_CU_REJECT_AURA_SAVING            = 0x10000000,
+    // unused                                    = 0x10000000,
     SPELL_ATTR0_CU_FORCE_AURA_SAVING             = 0x20000800,
+    SPELL_ATTR0_CU_ENCOUNTER_REWARD              = 0x40000000, // pussywizard
```

Comment:
SPELL_ATTR0_CU_ENCOUNTER_REWARD is not used in db so no need to have SQL for that